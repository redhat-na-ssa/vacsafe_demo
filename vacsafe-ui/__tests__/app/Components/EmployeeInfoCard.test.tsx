import React from 'react';
import { shallow } from 'enzyme';
import EmployeeInfoCard from '../../../src/app/Components/EmployeeInfoCard';

describe('<EmployeeInfoCard />', ()=> {
    it('should be defined', () => {
        expect(EmployeeInfoCard).toBeDefined()
    })
    it("should render without crashing", () => {
        const wrapper = shallow(<EmployeeInfoCard />)
        expect(wrapper)
    })
    it("should match existing snapshot", () => {
        const wrapper = shallow(<EmployeeInfoCard />)
        expect(wrapper).toMatchSnapshot()
    })
})
