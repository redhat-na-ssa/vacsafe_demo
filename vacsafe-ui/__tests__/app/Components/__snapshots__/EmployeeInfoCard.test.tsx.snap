// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmployeeInfoCard /> should match existing snapshot 1`] = `
<Card>
  <CardWithTitle
    title="Employee Information"
  />
  <CardBody>
    <Form>
      <FormGroup
        fieldId="profile-form-employee-id"
        label="Person ID"
      >
        <TextInput
          aria-label="profile-input-employee-id"
          isDisabled={true}
          type="text"
        />
      </FormGroup>
      <FormGroup
        fieldId="profile-form-employee-name"
        label="Name"
      >
        <TextInput
          aria-label="profile-input-employee-name"
          isDisabled={true}
          type="text"
        />
      </FormGroup>
      <FormGroup
        fieldId="profile-form-employee-dob"
        label="Date of Birth"
      >
        <TextInput
          aria-label="profile-input-employee-dob"
          isDisabled={true}
          type="text"
          value=""
        />
      </FormGroup>
      <FormGroup
        fieldId="profile-form-employee-email"
        label="Email"
      >
        <TextInput
          aria-label="profile-input-employee-email"
          isDisabled={true}
          type="text"
        />
      </FormGroup>
      <FormGroup
        fieldId="profile-form-agency"
        label="Agency"
      >
        <TextInput
          aria-label="profile-input-agency"
          isDisabled={true}
          type="text"
        />
      </FormGroup>
      <FormGroup
        fieldId="profile-form-division"
        label="Division"
      >
        <TextInput
          aria-label="profile-input-division"
          isDisabled={true}
          type="text"
        />
      </FormGroup>
    </Form>
  </CardBody>
</Card>
`;
